mlflow/models/evaluation/validation.py

```diff
@@ -0,0 +1,182 @@
+class MetricThreshold:
+    """
+    This class allows users to define metric thresholds for Model Validation. Allowed thresholds are
```

Comment:
Since this is a user-facing class, can we write this in the second person? I.e. "you" instead of "users"?